test/run_tests.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2.7
```

Comment:
Can you undo this commit. This change is not appropriate for this PR as it would affect other users. In the future any changes you make that are purely for your own system should not be commited.

Reply:
I just undo the commit.